man/systemd.net-naming-scheme.xml

```diff
@@ -362,11 +362,11 @@
         <varlistentry>
           <term><constant>v247</constant></term>
 
-          <listitem><para>If the PCI slot is associated with PCI bridge and that has multiple child network
+          <listitem><para>If the PCI slot is associated with PCI bridge, which has multiple child network
```

Comment:
with a PCI bridge that has multiple child network controllers, then ...

Reply:
I redid the whole paragraph, it needed that.